src/main/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.java

```diff
@@ -38,7 +38,7 @@
  * command does not show previous lines as changed.
  * </p>
  * <p>
- * Example (line 36 should not be in diff):
+ * Example:
```

Comment:
To me, I think it would be better to say `Example: (the line with 'No newline at end of file' should not be in the diff)`.

Reply:
Exactly! That's way better approach then